yaksh/test_models.py

```diff
@@ -838,17 +843,17 @@ def setUpClass(self):
 
         self.question1.language = 'python'
         self.question1.test_case_type = 'standardtestcase'
-        self.question1.summary = "Question1"
+        self.question1.summary = "Q1"
```

Comment:
I understand this problem you faced while writing testcases as the objects were modified!
I am slightly worried as the questions are fetched by ids at the top and not summary.

Reply:
Top where? I fixed all the problems that I encountered for this PR. Fixing it all through out `test_models` wouldn't be appropriate for this PR. Maybe in a future PR. Until then, maybe you can specify this particular issue in issue #360 .